src/mqt/predictor/rl/helper.py

```diff
@@ -421,3 +431,88 @@ def get_path_trained_model():
 
 def get_path_training_circuits():
     return get_path_training_data() / "training_circuits"
+
+
+def load_model(model_name: str):
+    path = get_path_trained_model()
+
+    if Path(path / (model_name + ".zip")).exists():
+        return MaskablePPO.load(path / (model_name + ".zip"))
+
+    print("Model does not exist. Try to retrieve suitable Model from GitHub...")
+    try:
+        mqtpredictor_module_version = metadata.version("mqt.predictor")
+    except Exception:
+        print(
+            "'mqt.predictor' is most likely not installed. Please run 'pip install . or pip install mqt.predictor'."
+        )
+        return False
+
+    version_found = False
+    response = requests.get("https://api.github.com/repos/cda-tum/mqtpredictor/tags")
+    available_versions = []
+    for elem in response.json():
+        available_versions.append(elem["name"])
+    print(available_versions, version.parse(mqtpredictor_module_version))
+    for possible_version in available_versions:
+        if version.parse(mqtpredictor_module_version) >= version.parse(
+            possible_version
+        ):
+            url = (
+                "https://api.github.com/repos/cda-tum/mqtpredictor/releases/tags/"
+                + possible_version
+            )
+            response = requests.get(url)
+            if not response:
+                print(
+                    "Suitable trained models cannot be downloaded since the GitHub API failed. "
+                    "One reasons could be that the limit of 60 API calls per hour and IP address is exceeded."
+                )
+                return False
+
+            response_json = response.json()
+            if "assets" in response_json:
+                assets = response_json["assets"]
+            elif "asset" in response_json:
+                assets = [response_json["asset"]]
+            else:
+                assets = []
+
+            for asset in assets:
+                if model_name in asset["name"]:
+                    version_found = True
+
+                if version_found:
+                    download_url = asset["browser_download_url"]
+                    print("Downloading model from: " + download_url)
+                    handle_downloading_model(download_url, model_name)
+                    break
+        if version_found:
+            break
+
+    if not version_found:
+        print("No suitable model found.")
+        return False
```

Comment:
This just seems wrong. The expected return value of this function is a model. Instead of returning a completely different type here, this should raise an Exception. Could be `InvalidArgument` or `RuntimeException` or something along those lines.

Reply:
Done.